source/graphics/GridGraphic.js

```diff
@@ -440,14 +482,15 @@ export default class GridGraphic extends Graphic {
       hexagonGrid.getTileEdge(),  // neighborhood radius
       2                           // min points per cluster
     )
-
     // return paths
     return clusters.map(clusterIndices => {
       const clusterPoints = clusterIndices.map(index => points[index])
-      return hull(
-        clusterPoints,
-        hexagonGrid.getTileEdge() // 'concavity', a.k.a. max edge length
-      )
+      return returnHull ?
+        hull(
```

Comment:
Can we push this `hull`-specific logic back up into `_computeOutlinePaths` so that `_computeClusters` just returns clusters? (Then we probably wouldn't need the internal `_clusters` helper.)